ansible_collections/arista/avd/roles/eos_designs/docs/input-variables.md

```diff
@@ -722,14 +722,14 @@ The following underlay routing protocols are supported:
 - EBGP (default for l3ls-evpn)
 - OSPF.
 - ISIS.
-- ISIS-SR*.
-- ISIS-LDP*.
-- ISIS-SR-LDP*.
-- OSPF-LDP*.
-- none**.
+- ISIS-SR¹.
+- ISIS-LDP¹.
+- ISIS-SR-LDP¹.
+- OSPF-LDP¹.
+- none².
 
-\* Only supported with core_interfaces data model.<br />
-\** For use with design type "l2ls" or other designs where there is no requirement for a routing protocol for underlay and/or overlay on l3 devices.
+¹ Only supported with core_interfaces data model.<br />
+² For use with design type "l2ls" or other designs where there is no requirement for a routing protocol for underlay and/or overlay on l3 devices.
```

Comment:
hmm the best we could do would be tooltips instead of these .. thoughts?

Reply:
We cannot use things like that, since it also needs to compile correctly in a YAML variant.

Reply:
this specific one  (and not the other in the schema) is not in yaml only md but still the only feature that exists would put the notes at the bottom of the humungous huge file so that would not work.

Reply:
ah missed that point, but yes it would just make it harder to maintain.